core/templates/dev/head/pages/landing/stewards/landing_page_stewards.html

```diff
@@ -544,7 +545,7 @@ <h2>
         margin-left: auto;
       }
       .oppia-stewards-button {
-        font-size: 1em;
+        font-size: 3.6vw;
```

Comment:
Hi @YashJipkate, can you please add a comment explaining the change.

Reply:
Actually, I used this value because the earlier value turned out to make the button (of which it is the text) overflow and I saw this value being used in the mobile view in the oppia.css file so thought it as a standard for mobile view font size for buttons. And also it solved the overflowing button problem.

Reply:
I added a comment anyways; Please let me know if it suffices. 
Thanks!

Reply:
Looks good, thanks!